packages/teleport/src/lib/tdp/client.ts

```diff
@@ -102,7 +103,7 @@ export default class Client extends EventEmitterWebAuthnSender {
     };
   }
 
-  processMessage(buffer: ArrayBuffer) {
+  async processMessage(buffer: ArrayBuffer) {
```

Comment:
now that this is an `async` function you should probably type the return value here.
```suggestion
  async processMessage(buffer: ArrayBuffer): Promise<void> {
```

Reply:
Why is that particularly important for `async` functions?

Reply:
async functions have an implicit return type of a `Promise` but the value from within that promise isn't automagically picked up by the compiler so by specifying the type in the generic there it just helps it be more explicit in the editor.

Reply:
> value from within that promise isn't automagically picked up by the compiler

it is actually:
<img width="487" alt="Screenshot 2022-08-09 at 18 13 52" src="https://user-images.githubusercontent.com/46656790/183703763-c656c85d-37e5-4166-a6ee-a9280b845955.png">

Reply:
Oh nice, it wasn't working for me in this PR so that's why I suggested the change.